ethereum/Strings.sol

```diff
@@ -10,8 +9,8 @@ library Strings {
      * @dev Converts a `uint256` to its ASCII `string` representation.
      */
     function toString(uint256 value) internal pure returns (string memory) {
-        // Inspired by OraclizeAPI's implementation - MIT licence
-        // https://github.com/oraclize/ethereum-api/blob/b42146b063c7d6ee1358846c198246239e9360e8/oraclizeAPI_0.4.25.sol
+    //     // Inspired by OraclizeAPI's implementation - MIT licence
```

Comment:
Weird double comment? It wouldn't usually matter but published contract code is permanent in IPFS.